simpa/core/device_digital_twins/illumination_geometries/ring_illumination.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-FileCopyrightText: 2021 Division of Intelligent Medical Systems, DKFZ
+# SPDX-FileCopyrightText: 2021 Janek Groehl
+# SPDX-License-Identifier: MIT
+
+import typing
```

Comment:
Please use collections.abc as typing is deprecated

Reply:
Please clarify what you mean. As far I know, typing is still commonly used nowadays (and collections.abc does not have Optional)

Reply:
I misread, I thought you were using it for more generic classes